GeneralUtilities/inc/PhiPrescalingParams.hh

```diff
@@ -10,9 +10,9 @@ namespace mu2e {
     float    _phase;
 
     struct Config {
-      fhicl::Atom<float> amplitude{fhicl::Name("amplitude"), fhicl::Comment("amplitude")};
-      fhicl::Atom<float> frequency{fhicl::Name("frequency"), fhicl::Comment("frequency")};
-      fhicl::Atom<float> phase    {fhicl::Name("phase"),     fhicl::Comment("phase")};
+      fhicl::Atom<float> amplitude{fhicl::Name("amplitude"), fhicl::Comment("amplitude"), 0.};
```

Comment:
Is the meaning of 0 that every event will be passed?  ( Just checking that understand the convention you have chosen).

We have a policy that fcl parameters that "affect physics" should not be given default values in code.   I would like to discuss with you and @brownd1978 if we should think of prescale factors as parameters that affect physics.

Reply:
These parameters are used for a specific case; prescaling the IPA electrons using their helices' `phi0` parameter. These parameters are not used if the `HelixSeedFilter_module` doesn't have the option `prescaleUsingD0Phi` set to `true`.  Prescaling based on this formula is supposed to provide a uniformly distributed helix population, which otherwise will be skewed following the muon beam orientation. Thus, I don't affect physics. 
Also, being forced to set these parameters in the trigger paths where they are not used would be confusing in my opinion.

Reply:
In that case I recommend the following.  Change their type from Atom<T> to OptionalAtom<T> and remove the default value from the code; in the comment text say that they are required when prescaleUsingD0Phi is true.  That way they do not clutter the fcl definitions in which they are not needed and they still obey our policy.

It's a good instinct not to clutter the fcl with irrelevant information.

Reply:
thanks for the suggestion. I just pushed the change.

Reply:
Thanks for getting this done.  Please watch for other targets of opportunity to use OptionalAtom.